nixos/modules/services/networking/nomad.nix

```
@@ -49,6 +49,18 @@ in
         '';
       };
 
+      settingsFiles = mkOption {
+        type = types.listOf (types.oneOf [ types.path types.str ]);
```

```suggestion
        type = types.listOf types.path;
```

I don't think paths relative to the working directory make sense, even if it's supported. Such an indirection seems counterproductive. Let's keep it simple.